#include <iostream>
using namespace std;

int main()
{
    throw "an exception of some type";
    return 0;
}
